worker/worker.go

```diff
@@ -415,21 +415,21 @@ func (w *Worker) slabMigrateHandler(jc jape.Context) {
 		}
 	}
 
-	// migrate the slab
-	numShardsMigrated, surchargeApplied, err := w.migrate(ctx, slab, up.ContractSet, dlContracts, ulContracts, up.CurrentHeight)
+	// migrate the slab and handle alerts
+	err = w.migrate(ctx, slab, up.ContractSet, dlContracts, ulContracts, up.CurrentHeight)
 	if err != nil {
-		jc.Encode(api.MigrateSlabResponse{
-			NumShardsMigrated: numShardsMigrated,
-			SurchargeApplied:  surchargeApplied,
```

Comment:
I might be missing it but are we no longer alerting of migrations with applied surcharge or successful migrations and how many slabs were migrated?

Reply:
I did that on purpose. We added both of those for debugging purposes (in a way) and I wanted to propose removing them. I had the critical migration succeeded alert on my node a couple of times, so the surcharge system occasionally triggers. The alert is not useful I think, especially because you have to manually resolve it, we never dismiss it. I don't know if you saw but I extended the `hint` in the migration failed alert to include a piece about the multiplier.

Reply:
I agree that we probably don't need the number of shards in the alerts or regular logging. But I'd make sure we have some debug logging mentioning it.
The surcharge system is a bit more critical. In theory it never triggers but if it starts triggering people should be aware of it to be able to adjust their gouging settings. The only time I ever saw is was on Arequipa which was in a pretty bad shape already.

Reply:
Ok. I added debug logging in the worker. The `surchargeApplied` was still debug logged somewhere I believe, but now it is very clearly and explicitly logged. Same with the num shards migrated.